spec/requests/api/v1/members_spec.rb

```diff
@@ -0,0 +1,152 @@
+require "rails_helper"
+
+RSpec.describe "Api::V1::Members", type: :request do
+  let(:role) { FactoryBot.create(:role) }
```

Comment:
No es necesario invovar a la clase `FactoryBot` porque ya existe el factory.
```suggestion
  let(:role) { create(:role) }
```